extensions/placeholder-data-source/src/main/resources/META-INF/services/org.eclipse.edc.spi.system.ServiceExtension

```diff
@@ -0,0 +1 @@
+de.sovity.edc.extension.placeholderdatasource.PlaceholderDataSourceExtension
```

Comment:
my recommendation:

the interface required for the wrapper-common-mappers lies there
the endpoint implementation should be registered in the wrapper extension

there's no need for a separate extension, because a separate extension really needs to be justified (use case for not using it, e.g.)

Reply:
Config added to `launchers/.env.connector`

Reply:
> the endpoint implementation should be registered in the wrapper extension

I registered it, but only for tests as it's now not injected anymore.